sormas-e2e-tests/src/test/java/org/sormas/e2etests/helpers/WebDriverHelpers.java

```diff
@@ -237,6 +248,26 @@ public void waitUntilIdentifiedElementIsPresent(final By selector) {
         () -> assertThat(getNumberOfElements(selector) > 0).isTrue());
   }
 
+  public String getTextOfSelectedWebElementFromList(String mainDivID) {
```

Comment:
We need to remove this method, because if something is changed in UI, the `id`  and is added the class we will need to change everything.